lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -756,6 +756,50 @@ static LogicalResult verifyWhileOp(WhileOp whileOp) {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// Calyx library ops
+//===----------------------------------------------------------------------===//
+
+// The Calyx library operations implement the OpAsmOpInterface which expects a
+// getAsmResultNames member function to be defined out of line of the
+// tablegen'erated files. Since there is no "extraClassDefinitions" function in
+// the TableGen backend for Op classes, we have to manually define these here.
+static void getBinOpAsmResultNames(Operation *op,
+                                   OpAsmSetValueNameFn setNameFn) {
+  unsigned numResults = op->getNumResults();
+  assert(numResults == 3);
+  SmallVector<StringRef> portNames;
+  SmallVector<SmallString<8>> inNames;
+  for (size_t i = 0, e = numResults - 1; i != e; ++i) {
+    inNames.emplace_back("in" + std::to_string(i));
```

Comment:
Nit: `lhs`, `rhs` to follow Combinational dialect convention, or `left`, `right` to be closer to Calyx convention.